modules/vaos/app/services/vaos/v2/appointments_service.rb

```diff
@@ -570,6 +573,15 @@ def page_params(pagination_params)
       def date_format(date)
         date.strftime('%Y-%m-%dT%TZ')
       end
+
```

Comment:
I wanted to use faraday's plug system to insert this functionality, but it wasn't ideal because the configuration is for all requests within the service, not separated by endpoint. So getting all the data you need into the function would require more substantial changes and I didn't think it was worth it.